windows-app-samples/README.md

```diff
@@ -0,0 +1,15 @@
+# WindowsAppSamples
+
+Windows apps can be scanned for accessibility issues using the tools in the [Axe.Windows](https://github.com/microsoft/axe-windows) repo. These tools do not currently produce Sarif files. The general pattern is as follows:
+
+- Use your favorite framework to get your application to the state that needs to be scanned.
```

Comment:
It might be nice to offer some examples here (eg, [Appium with WinAppDriver](https://github.com/appium/appium/blob/master/docs/en/drivers/windows.md))

Reply:
Done